addons/resource.language.en_gb/resources/strings.po

```diff
@@ -7185,7 +7185,30 @@ msgctxt "#13461"
 msgid "Enable this option to use hardware acceleration for the HEVC codec. If disabled the CPU will be used instead."
 msgstr ""
 
-#empty strings from id 13462 to 13504
+#: system/settings/settings.xml
+msgctxt "#13462"
+msgid "PRIME Renderer"
+msgstr ""
+
+#. Description of setting with label #13462 "Use Prime Renderer"
+#: system/settings/settings.xml
+msgctxt "#13463"
+msgid "This option switches between Direct and OpenGLES Rendering."
+msgstr ""
+
+#. adjust refreshrate
```

Comment:
I don't get how `adjust refreshrate` relates to this label. Could you be a bit more specific here for the contextual comment? As a translator I would not know how to correctly translate `Direct` here. Is it an actual name of a rendering path of the Prime renderer so that it must not be translated or could it be translated literally etc etc.

Reply:
Sorry, this was just an afterthought. I've updated them properly now. Hopefully it makes more sense.